core/schemas/task.py

```diff
@@ -18,6 +16,9 @@
 from core.schemas.model import YetiModel
 from core.schemas.observable import Observable, ObservableType
 from core.schemas.template import Template
+from core.clients import file_storage
+
+file_storage_client = file_storage.get_client(yeti_config.get("system", "export_path", "/opt/yeti/exports"))
```

Comment:
let's use uppercase for globals

Reply:
Fixed in [e933f19](https://github.com/yeti-platform/yeti/pull/1137/commits/e933f196fb13f22bd65a6c1cba03ce3d9b1eb749)